Dockerfile

```diff
@@ -22,17 +19,15 @@ RUN if [ "${BUILD_ENV}" = "development" ] ; then \
 RUN npm install --global sfdx-cli --ignore-scripts
 
 # JS client setup:
-COPY ./package.json /app/package.json
-COPY ./yarn.lock /app/yarn.lock
+COPY ./package.json package.json
+COPY ./yarn.lock yarn.lock
+RUN yarn install --check-files
 
 COPY . /app
 
-# === Actually run things:
-WORKDIR /app
-RUN yarn install --check-files
-
```

Comment:
The biggest change I made was moving `yarn install` before `COPY . /app`. This should help Docker skip re-installing frontend dependencies when source code changes without affecting dependencies.